frontend_app_admin_workspace_user/i18next.scanner/en/translation.json

```diff
@@ -36,14 +37,21 @@
   "Password": "Password",
   "Create the user": "Create the user",
   "Create a user": "Create a user",
-  "Please type a name, an email, a password and select a profile": "Please type a name, an email, a password and select a profile",
+  "Please enter a name, a password, select a profile, and at least one between username or email": "Please enter a name, a password, select a profile, and at least one between username or email",
   "Please set a password": "Please set a password",
   "Full name": "Full name",
   "Full name must be at least 3 characters": "Full name must be at least 3 characters",
+  "Username must be at least 3 characters": "Username must be at least 3 characters",
+  "Username can't contain any whitespace": "Username can't contain any whitespace",
   "Email notification are disabled, please manually notify users of any change": "Email notification are disabled, please manually notify users of any change",
   "Error, invalid email address": "Error, invalid email address",
+  "Error while checking username availability": "Error while checking username availability",
   "Create a shared space": "Create a shared space",
   "Home": "Home",
   "Users": "Users",
-  "Administration": "Administration"
+  "Administration": "Administration",
+  "Username": "Username",
+  "@username": "@username",
+  "The user will not be able to reset his password by himself without an email linked to his account.": "The user will not be able to reset his password by himself without an email linked to his account.",
```

Comment:
We probably should use a gender neutral pronoun (singular "they", "their"), or a formulation that avoids using any masculine form. We can discuss this.

Reply:
I love this idea, if you have any suggestion of a phrase that doesn't use any male pronoun, it would be even better, since neither Portuguese nor French have a neutral pronoun (yet).

Reply:
Proposals:
- Linking an email address is required for the user to be able to reset the password.
- The user must link an email address is required to be able to reset the password.
- The user must link an email address is required to be able to reset their password [by themself].

Unfortunately, I guess we probably cannot avoid gendered pronoun for every language. Translation will probably need to use different tricks so it is probably okay to use singular they in English and use other tricks like periphrases if they are not too lengthy. The translation does not need to be a perfect, literal translation anyway.

I agree with you though. If we can do without singular they for most strings it's probably even better and it might force us to find elegant formulations :-)

Reply:
Yes, I totally agree with you. I talked to @celiamgtt and we liked the option better: 
`Linking an email address is required for the user to be able to reset the password.`
I think we'll adopt this one, besides being neutral it's more explicit and clean than the previous one. 
Thanks a lot for the suggestions, I'll adapt the issue description as well.

Reply:
Changed

Reply:
That was my preferred option as well :-)